### DISCUSSION TITLE
Rss feed

### DISCUSSION THREAD

**Discussion by TeckboyAj** [FreshRSS/FreshRSS#5573] [Q&A] [answered]
I want to ask is it possible that the content which we have published on the fressrss, we can export all the post.in form of xml file. Is it possible?

**Comment by Alkarex** [comment]
Hello,
Have you tried the feed button? Any FreshRSS view can be seen as RSS (XML)
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/c3dff011-a6e2-4b08-a31d-36be08aba43e)

* Since X hours: `https://freshrss.example/i/?a=rss&hours=3`
* Explicit number: `https://freshrss.example/i/?a=rss&nb=10`
* Limited by `min_posts_per_rss` and `max_posts_per_rss` in user config

**Reply by TeckboyAj** [reply]
Yes, I tried but this is not working in my server.

**Reply by Alkarex** [reply]
What do you mean more precisely by *not working*?

**Reply by TeckboyAj** [reply]
When I click on feed but one file is downloaded not getting the url of the feed.

**Reply by Frenzie** [reply]
Right click, copy?

**Reply by thoresson** [reply]
![CleanShot 2024-12-20 at 09 12 46](https://github.com/user-attachments/assets/8f01606f-0bd2-4c4e-acc5-7e4b2f5d040d)

I don't have the RSS option in my button bar, and can't find a setting or a plugin that would enable it. What am I missing? :)

**Reply by Alkarex** [reply, marked as answer]
This old answer higher up is not valid anymore.
There is a better and safer mechanism now, by sharing user queries. See https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries
